### PR TITLE
Feature/update readme

### PR DESCRIPTION
**Pull request by billkremer**
Uploaded the new file and fixed a small spelling error

### CODE REVIEW

File: helloworld.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Hello World this is Earth
+Thanks
+
```
Comment (kremerben) [inline helloworld.txt]: LGTM

File: helloworld.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Hello World this is Earth
```
Comment (kremerben) [inline helloworld.txt]: Should this be capitalized?

Comment (billkremer) [inline helloworld.txt]: no, you're wrong

Comment (kremerben) [inline helloworld.txt]: ok cool, just checking

**Pull request merged**